uber/templates/macros.html

```diff
@@ -82,6 +82,36 @@
 {%- endmacro %}
 
 
+{% macro form_group_scaffolding(
+    label=False,
+    help=False,
+    is_required=False,
+    size=6,
+    form_class='',
+    static_wrap=False) -%}
+    <div class="form-group {{ form_class }}">
+        <label class="col-sm-3 control-label{% if not is_required %} optional-field{% endif %}">
+            {{ label }}
+        </label>
+    <div class="col-sm-{{ size }}">
+        {% if static_wrap %}
+            <div class="form-control-static">
+                {{ caller() }}
+            </div>
+        {% else %}
+            {{ caller() }}
+        {% endif %}
+    </div>
+    {% if help %}
+        <div class="clearfix"></div>
+        <p class="col-sm-{{ size }} col-sm-offset-3 help-block">
```

Comment:
Here we have a stylistic choice which I don't think we've discussed? I prefer `help-block`s to have `col-sm-9`, so they're allowed to extend beyond the length of the field itself. I always think it looks silly when the text prematurely wraps. However, before I started working on this project, all the `help-block`s were constrained to the same width as the field. I could be persuaded to return to that style, but whatever we do, we should be consistent about it.

If we DO return to constraining the `help-block` width, then we can remove the `clearfix` and put the help block back under the `col-sm-{{ size }}` div.